code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -1352,12 +1352,11 @@
 
 /obj/item/integrated_circuit/input/tcomm_interceptor/Destroy()
 	qdel(receiver)
-	GLOB.ic_jammers -= src
 	..()
```

Comment:
At least `Cut()` the `encryption_keys` list. If you feel brave, `QDEL_LIST` it or just let it drop all the stored encryption keys.